### PR TITLE
fix bug indexer

### PR DESCRIPTION
**Pull request by 0xBahamoot**

### CODE REVIEW

File: otaindexer/master.go [hunk 1]
```diff
@@ -164,24 +164,24 @@ var OTAAssignChn chan OTAAssignRequest
...
+		if d == 100 {
+			wg.Wait()
+			d = 0
+		}
+		wg.Add(1)
+		d++
+		go func(key *OTAkeyInfo) {
+			err := tempFixOTAKey(key.OTAKey, key.Pubkey)
+			if err != nil {
+				panic(err)
```
Comment (0xakk0r0kamui) [inline otaindexer/master.go:179]: Remove panic, cuz it may block wg.Wait()

Comment (0xakk0r0kamui) [reply otaindexer/master.go:179]: Using fan in fan out is better?

File: otaindexer/master.go [hunk 2]
```diff
@@ -164,24 +164,24 @@ var OTAAssignChn chan OTAAssignRequest
...
+			}
+			wg.Done()
+		}(v)
+	}
+	wg.Wait()
```
Comment (0xakk0r0kamui) [inline otaindexer/master.go:184]: Should we create fixed worker and run?

Comment (0xBahamoot) [reply otaindexer/master.go]: fixed 100 at a time

**Review by 0xakk0r0kamui — APPROVED** [review]

**Pull request closed without merging**